extensionsGallery-insider.json

```diff
@@ -3822,15 +3822,72 @@
 					],
 					"statistics": [],
 					"flags": ""
-				}
+				},
+				{
+					"extensionId": "78",
+					"extensionName": "extraSqlScriptAs",
```

Comment:
Naming convention uses dash-spaced for extension names, e.g. `extra-sql-script-as`. Could you change that both here and in the source/vsix?